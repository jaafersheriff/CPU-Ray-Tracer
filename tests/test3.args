pixelray simple.pov 640 480 490 120
