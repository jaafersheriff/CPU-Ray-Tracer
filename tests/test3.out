Pixel: [490, 120] Ray: {0 0 14} -> {0.3259 -0.2284 -0.9174}
